### ISSUE TITLE
Open Quickly improvements

### ISSUE THREAD

**Issue by ottob** [jakob/Postico#258]
I have many copies of a database. It would be nice if I could see which one I will navigate to.
<img width="1331" alt="screen shot 2016-08-22 at 09 30 40" src="https://cloud.githubusercontent.com/assets/67104/17846858/434992f4-684b-11e6-8744-0aecfb9550e0.png">

Perhaps some kind of heuristic/statistics could be used so tables that are browsed more often appear at the top of the list?

**Comment by jakob** [comment]
Hey Otto!

Thanks for the screenshot; I haven't considered the use case where you have a number of very similar databases. I've now improved the result list to also display the database of the matched objects.

While working on the Open Quickly command, I also discovered a few performance issues. I optimised the filtering algorithm. “Open Quickly” should now be a lot quicker!

Finally, to make search results more relevant, I tweaked some of the scoring parameters. I also rank recently used tables higher, as well as tables from the current database. I'm not really sure if I found the optimal value for all the parameters, but I hope that it works a bit better now.

Anyway, here's the link to a new build. Would love to hear your comments:
https://eggerapps-downloads.s3-eu-west-1.amazonaws.com/postico-1505.zip
